locales/pl-PL.json

```diff
@@ -82,7 +82,7 @@
   "Name": "Nazwa",
   "Username": "Nazwa użytkownika",
   "Password": "Hasło",
-  "Email": "E-mail",
+  "Email": "E-Mail",
```

Comment:
Hey @zlElo what's the point of this fix? I don't think I've ever seen this spelling with a capital `M`. It's usually either `E-mail` or `e-mail`

Reply:
In Germany, we write every time E-Mail instead of E-mail.

Reply:
Right, so German locale has it spelled correctly https://github.com/swiftyapp/swifty/blob/master/locales/de-DE.json#L85 
Why changing Polish locale?

Reply:
I know, I translated it into German 😂 But Okay, I understand... Then I close it